blueprints/ember-cli-horizon/index.js

```diff
@@ -1,13 +1,24 @@
 /*jshint node:true*/
 module.exports = {
-    description: 'Default blueprint to be run after install.',
-
-    afterInstall: function() {
-        return this.addPackagesToProject([
-            { package: '@horizon/client' },
-            { package: 'bufferutil' },
```

Comment:
Ember-browserify is how the addon is actually pulling in the npm package in. It seems to be the only good way to bring in npm dependencies right now. As for those other two: https://github.com/ef4/ember-browserify/issues/86